## 89% of end-end tests have passed
```
Total executed tests: 62
Failed tests: 6
Expected to fail tests: 1
Passed tests: 55
```
```
IDs of failed tests: 702845,702786,741054,702809,702948,703629 
```
```
IDs of expected to fail tests: 702844 
```
<h3>Failed tests (6)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feat/pre_login_log&PR_ID=21987&APK_NAME=21987.apk&TR_CASE_IDS=702845,702786,741054,702809,702948,703629">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788002">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:887: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:341: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787991">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:887: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:341: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESqcGmse0gq6Jh03UW">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVgAvO7jNt1JMYrX3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787994">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:887: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:341: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788011">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:887: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:341: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788024">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:887: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:341: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788035">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping left on element Button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Imported account')]` </div></blockquote></p>

```
critical/test_fallback.py:212: in test_fallback_add_key_pair
    wallet_1.get_account_element(account_name=imported_key_pair_account_name).swipe_left_on_element()
../views/base_element.py:281: in swipe_left_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:274: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Imported account')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788001">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'https://m.youtube.com/watch?v=Je7yErjEVt4')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-4']/android.widget.TextView[2]` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:662: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: No preview is loaded for url https://youtu.be/Je7yErjEVt4
E    Device 1: No preview is loaded for url https://www.youtube.com/watch?v=XN-SVmuJH2g&list=PLbrz7IuP1hrgNtYe9g6YHwHO6F3OqNMao
E    Device 1: No preview is loaded for url https://m.youtube.com/watch?v=Je7yErjEVt4 
```

<b>[[Youtube links preview is not loaded on LambdaTest emulators, needs investigation]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (55)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788013">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuaRHAQrjIHxqtuud">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESixCGcvZCj2Qu2fTn">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788012">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuaRHAQrjIHxqtuud">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESixCGcvZCj2Qu2fTn">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788041">test_activity_centre_contact_request_receiver_offline</a>, id: 741809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuaRHAQrjIHxqtuud">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESixCGcvZCj2Qu2fTn">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788015">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787984">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788003">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788008">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788019">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788023">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQv2TSMWIdlosj1B5">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788025">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwlRZDOFHCn6zRIXO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787986">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwlRZDOFHCn6zRIXO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788030">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwlRZDOFHCn6zRIXO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788022">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStPwhFddBEhsKySKE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrXTkAYTpTlNBNR4X">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787989">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStPwhFddBEhsKySKE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrXTkAYTpTlNBNR4X">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787990">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStPwhFddBEhsKySKE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrXTkAYTpTlNBNR4X">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788029">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788037">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788034">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788038">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788036">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788005">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6eSE0G5twbDVPoZD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6N9U9RIriPd2jWCg">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788004">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6eSE0G5twbDVPoZD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6N9U9RIriPd2jWCg">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787987">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6eSE0G5twbDVPoZD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6N9U9RIriPd2jWCg">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788017">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787992">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787993">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787982">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788018">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788021">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpwSCIjYvHuS8jsbY">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCPdAUR8l9F0fA216">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkzbNH9W09UzR23ut">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788031">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788032">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788033">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestAndroid12:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788039">test_create_account_android_12</a>, id: 741806 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScB7J4AS7eTENuA5l">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788028">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788027">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788006">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787980">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787985">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787981">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787983">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787995">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787988">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788020">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLOUMBFh0Ahjqapfy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESP1MroWsMYvyXEvl0">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787998">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788009">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788014">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787999">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787997">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3787996">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788026">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788007">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788000">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788016">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8pG7WEFV73FZqNvM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS5AbHnrLQ091T9h2">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestAndroid13:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788040">test_create_account_android_13</a>, id: 741807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESvimLmsQ6Du9Vehug">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3788010">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZOi6HHL1bzMJJqO7">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESa9icOKXY3R8MM8Qm">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/module/LogManager.kt
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/module/LogManager.kt
@@ -35,6 +35,11 @@ class LogManager(private val reactContext: ReactApplicationContext) : ReactConte
         return File(pubDirectory, gethLogFileName)
     }
 
+    private fun getPreLoginLogFile(): File {
+        val pubDirectory = utils.getPublicStorageDirectory()
+        return File(pubDirectory, preLoginLogFileName)
+    }
+
     fun prepareLogsFile(context: Context): File? {
         val logFile = getGethLogFile()
 
@@ -156,6 +161,7 @@ class LogManager(private val reactContext: ReactApplicationContext) : ReactConte
         val statusLogFile = File(logsTempDir, statusLogFileName)
         val gethLogFile = getGethLogFile()
         val requestLogFile = getRequestLogFile()
+        val preLoginLogFile = getPreLoginLogFile()
 
         try {
             if (zipFile.exists() || zipFile.createNewFile()) {
@@ -174,6 +180,9 @@ class LogManager(private val reactContext: ReactApplicationContext) : ReactConte
             val filesToZip = mutableListOf(dbFile, gethLogFile, statusLogFile)
             if (requestLogFile.exists()) {
                 filesToZip.add(requestLogFile)
+            }
+            if (preLoginLogFile.exists()) {
+                filesToZip.add(preLoginLogFile)
             }
             val zipped = zip(filesToZip.toTypedArray(), zipFile, errorList)
             if (zipped && zipFile.exists()) {
@@ -205,6 +214,7 @@ class LogManager(private val reactContext: ReactApplicationContext) : ReactConte
         private const val gethLogFileName = "geth.log"
         private const val statusLogFileName = "Status.log"
         private const val requestsLogFileName = "api.log"
+        private const val preLoginLogFileName = "pre_login.log"
         private const val logsZipFileName = "Status-debug-logs.zip"
     }
 }

--- a/modules/react-native-status/ios/RCTStatus/LogManager.m
+++ b/modules/react-native-status/ios/RCTStatus/LogManager.m
@@ -35,7 +35,7 @@ RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
 
     NSURL *mainGethLogsFile = [rootUrl URLByAppendingPathComponent:@"geth.log"];
     NSURL *mainLogsFile = [logsFolderName URLByAppendingPathComponent:@"geth.log"];
-    NSURL *preLoginLogFile = [logsFolderName URLByAppendingPathComponent:@"pre_login.log"];
+    NSURL *preLoginLogFile = [rootUrl URLByAppendingPathComponent:@"pre_login.log"];
 
     NSURL *requestsLogFile = [rootUrl URLByAppendingPathComponent:@"api.log"];
 
@@ -49,7 +49,7 @@ RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
     }
     
     if ([fileManager fileExistsAtPath:preLoginLogFile.path]) {
-        [fileManager copyItemAtPath:preLoginLogFile.path toPath:preLoginLogFile.path error:nil];
+        [fileManager copyItemAtPath:preLoginLogFile.path toPath:[logsFolderName URLByAppendingPathComponent:@"pre_login.log"].path error:nil];
     }
 
     [SSZipArchive createZipFileAtPath:zipFile.path withContentsOfDirectory:logsFolderName.path];

--- a/modules/react-native-status/ios/RCTStatus/LogManager.m
+++ b/modules/react-native-status/ios/RCTStatus/LogManager.m
@@ -35,6 +35,7 @@ RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
 
     NSURL *mainGethLogsFile = [rootUrl URLByAppendingPathComponent:@"geth.log"];
     NSURL *mainLogsFile = [logsFolderName URLByAppendingPathComponent:@"geth.log"];
+    NSURL *preLoginLogFile = [logsFolderName URLByAppendingPathComponent:@"pre_login.log"];
 
     NSURL *requestsLogFile = [rootUrl URLByAppendingPathComponent:@"api.log"];
 
@@ -45,6 +46,10 @@ RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
     
     if ([fileManager fileExistsAtPath:requestsLogFile.path]) {
         [fileManager copyItemAtPath:requestsLogFile.path toPath:[logsFolderName URLByAppendingPathComponent:@"api.log"].path error:nil];
+    }
+    
+    if ([fileManager fileExistsAtPath:preLoginLogFile.path]) {
+        [fileManager copyItemAtPath:preLoginLogFile.path toPath:preLoginLogFile.path error:nil];
     }
 
     [SSZipArchive createZipFileAtPath:zipFile.path withContentsOfDirectory:logsFolderName.path];

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "feat/pre_login_log",
-    "commit-sha1": "7ae58a47707614ae83122f2a686baf1aefedfe6b",
-    "src-sha256": "1mdsynbc3sifm8lh9h7ih6x8wpznxrxx283f1w19ik2blviwq1a5"
+    "version": "v10.6.0",
+    "commit-sha1": "c5e4eb1a56ea7641efcc7cc677be7fd5d3f92225",
+    "src-sha256": "0n8x03c4dqxwcl27y8x6hmn6wbfpq3llkj15hhjxc4baaqwrhy8i"
 }

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v10.5.0",
-    "commit-sha1": "5a7969593c98cd7906607042ee19c2386a673ec3",
-    "src-sha256": "0kqll98235p2nxkccvp0asrk9w5k0zmd7mywparnjfxv5cav8qmb"
+    "version": "feat/pre_login_log",
+    "commit-sha1": "7ae58a47707614ae83122f2a686baf1aefedfe6b",
+    "src-sha256": "1mdsynbc3sifm8lh9h7ih6x8wpznxrxx283f1w19ik2blviwq1a5"
 }

